terraform-plans/templates/github/charm_check.yaml.tftpl

```diff
@@ -116,11 +116,15 @@ jobs:
             echo "TEST_MODEL_CONSTRAINTS=arch=arm64" >> "$GITHUB_ENV"
           fi
 
+      - name: Build the charm
+        run: chharmcraft -v pack
+
       - name: Run tests
         run: $${{ matrix.test-command }}
         env:
           TEST_JUJU3: "1"  # https://github.com/openstack-charmers/zaza/pull/653
           TEST_JUJU_CHANNEL: $${{ matrix.juju-channel }}
+          CHARM_PATH: "$${{ github.workspace}}/$${{ matrix.charm-file }}"
```

Comment:
The question here is do we autodetect the charm file, or do we include it in the workflow matrix?

Reply:
Autodetecting for now, because all our charms only output a single charm file by default.  We can revisit later.